r/R/dataset-format.R

```diff
@@ -53,7 +53,7 @@
 #' It returns the appropriate subclass of `FileFormat` (e.g. `ParquetFileFormat`)
 #' @rdname FileFormat
 #' @name FileFormat
-#' @examplesIf arrow_with_dataset()
+#' @examplesIf arrow_with_dataset() && tolower(Sys.info()[["sysname"]]) == "windows"
```

Comment:
Oops, this is run-if, not skip-if

```suggestion
#' @examplesIf arrow_with_dataset() && tolower(Sys.info()[["sysname"]]) != "windows"
```